src/test/java/io/github/glytching/junit/extension/random/RandomBeansExtensionFieldTest.java

```diff
@@ -80,6 +86,11 @@ public void canInjectAFullyPopulatedRandomObject() {
     assertThatDomainObjectIsFullyPopulated(fullyPopulatedDomainObject);
   }
 
+  @Test
+  public void acnInjectStaticFields() {
```

Comment:
Minor typo here: should be `canInjectStaticFields`.

Reply:
Done! Thanks, I good at making typos 😅